go.mod

```diff
@@ -0,0 +1,128 @@
+module volcano.sh/volcano-global
+
+go 1.21.10
+
+require (
+	github.com/karmada-io/karmada v1.10.2
+	github.com/spf13/pflag v1.0.5
+	gomodules.xyz/jsonpatch/v2 v2.4.0
+	k8s.io/api v0.29.4
+	k8s.io/apimachinery v0.29.4
+	k8s.io/client-go v0.29.4
+	k8s.io/component-base v0.29.4
+	k8s.io/klog/v2 v2.120.1
+	k8s.io/kube-openapi v0.0.0-20240228011516-70dd3763d340
+	sigs.k8s.io/controller-runtime v0.17.5
+	volcano.sh/apis v1.9.0
+	volcano.sh/volcano v1.9.0
```

Comment:
Change to v1.10 is better and what's the latest k8s version does Karmada support?

Reply:
If it's not used we should exec `go mod tidy` to remove them: )